runtime/thunk/thunk.ts

```diff
@@ -198,6 +212,13 @@ function findParentFormula(
   }
 }
 
+export function ensureExists<T>(value: T | null | undefined, message?: string): T {
+  if (typeof value === 'undefined' || value === null) {
+    throw new Error(message || `Expected value for ${String(value)}`);
+  }
+  return value;
+}
+
 export function ensureSwitchUnreachable(value: never): never {
   throw new Error(`Unreachable code hit with value ${String(value)}`);
 }
```

Comment:
these methods probably don't need to be exported. 

I recall this method was here instead being imported to avoid dependencies in the old days. it's probably okay to simply import them now?

Reply:
Yeah, don't need to export, you're right.  I haven't kept up to date on what dependencies are safe w.r.t. the thunk bundle, etc.

Reply:
yeah I was not sure but I saw it's importing from '../../helpers/migration' now.